subprojects/packagefiles/libserialport/meson.build

```diff
@@ -0,0 +1,48 @@
+project(
+  'libserialport',
+  'c',
+  version: '0.1.2',
+  license: 'LGPL3+',
+  meson_version: '>= 0.47.0',
+)
+
+libserialport_includes = include_directories(['.'])
+
+libserialport_headers = files('libserialport.h')
+
+libserialport_sources = files('serialport.c', 'timing.c')
+
+libserialport_cflags = []
+libserialport_ldflags = []
+
+if host_machine.system() == 'linux'
+  libserialport_sources += files('linux.c', 'linux_termios.c')
+  libserialport_cflags += '-DLIBSERIALPORT_ATBUILD'
+elif host_machine.system() == 'windows'
+  libserialport_sources += files('windows.c')
+  libserialport_cflags += '-DLIBSERIALPORT_MSBUILD'
+  libserialport_ldflags += '-lsetupapi'
+elif host_machine.system() == 'darwin'
+  libserialport_sources += files('macosx.c')
+  libserialport_cflags += '-DLIBSERIALPORT_ATBUILD'
+elif host_machine.system() == 'freebsd'
+  libserialport_sources += files('freebsd.c')
+  libserialport_cflags += '-DLIBSERIALPORT_ATBUILD'
+endif
+
+libserialport_lib = library(
+  'libserialport',
+  libserialport_sources,
+  c_args: libserialport_cflags,
+  link_args: libserialport_ldflags,
+  include_directories: libserialport_includes,
+  version: meson.project_version(),
```

Comment:
Upstream uses a libtool version distinct from the project version, i.e. `SP_LIB_VERSION_STRING` is `1:0:1` which corresponds to 0.1.0.